.github/workflows/build.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: mv ./xk6-neofs* ./xk6-neofs-${{ matrix.os.bin-name }}-${{ matrix.arch }}
 
       - name: Upload artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
I found out that a new version of the `upload-artifact` action was released. However, as noted in [https://github.com/actions/upload-artifact?tab=readme#v4---whats-new](https://github.com/actions/upload-artifact?tab=readme#v4---whats-new), it cannot be used on GHES. I don't know if this is relevant for us. If not, I'd prefer to split this commit from the previous one.

Reply:
GHES is irrelevant for us and IIRC we have this version working already in other repositories.

Reply:
Ok, I've split these commits.